src/css/styles.scss

```diff
@@ -234,35 +238,35 @@ main {
   align-content: flex-start;
   flex-direction: row;
```

Comment:
Maybe we should note the default font in here to dry things up a bit? Not sure if it would actually be that helpful, but just a thought